Workbooks/Azure Monitor - Workspaces/Agent Health/Overview/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Log Analytics Agent Health",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "workbook", "resourceType": "microsoft.operationalinsights/workspaces", "order": 100 }
```

Comment:
Are the workbooks in 'Workspace' the same? If they are, maybe we don't modify/add any workbooks there until we need to make the resource parameter changes. Instead we can add the gallery 
`{ "type": "workbook", "resourceType": "Azure Monitor", "order": 100 }` to this gallery array. That way we don't need to maintain two copies of the same templates and this template would show up on both Azure Monitor as well as the workspace resource.

Because when I tried previewing what this would look like using a storage account, I got some weird behavior. For example when I go into the workspace, it looks like I have two of the same workbooks here:

![image](https://user-images.githubusercontent.com/7664931/88122307-1ca2c000-cb7d-11ea-8a40-7e1282816445.png)

Reply:
We just fixed this during the Teams call.